card/lib/card/cache.rb

```diff
@@ -106,7 +106,7 @@ def reset_soft
       # (the non-standard caches)
       def reset_other
         Card::Codename.reset_cache
-        Cardio.delete_tmp_files!
+        ::Cardio.delete_tmp_files!
```

Comment:
these are the ones I don't see value in.

Reply:
I think I have the loading of the CardClassMethods on Card slightly wrong. I think it should do 'Card.extend ...' in Cardio where it loads mods on the load hook.

It's like Card::Cardio gets defined empty and catches these. I think this is in Card.rb, but not sure yet.